ozone-distro-archetype/src/main/resources/archetype-resources/readme/impl-guide.md

```diff
@@ -0,0 +1,60 @@
+#  Ozone ${distributionName} - Implementer Guide
+
+This distribution can be run using the Ozone Docker project, which is the default configuration for this. The quick start command below is for demonstration and trial purposes and would not be suitable for a stable environment.
+
+## Quick Start
+
+Build
+```bash
+./mvnw clean package
+```
+
+Run
+```bash
+source target/go-to-scripts-dir.sh
+./start-ozone.sh
+```
+
+### Working on configurations:
+
+If needed to work on the distro configurations and see the results, you have several options:
+- (1) Turn down the whole project with its volumes, build again and run.
+- (2) Replace files in the mounted Docker volume (all files or only individual files)
+
+#### Option 1. Turn down the whole project and start afresh
+```bash
+source target/go-to-scripts-dir.sh
+./destroy-demo.sh
+```
+
+Re-build:
+```bash
+./scripts/mvnw clean package
```

Comment:
This seems to assume the Maven Wrapper is in **scripts/** though, isn't it?

Reply:
I moved it and updated the text here.